src/test/java/seedu/duke/command/CommandAddClientTest.java

```diff
@@ -54,36 +54,32 @@ public class CommandAddClientTest {
     public PairingList pairingList = new PairingList();
     public Storage storage = new Storage(clientList, propertyList, pairingList);
 
+    public static final int OFFSET_UNIT_VALUE = 1;
+
     @Test
-    public void execute() {
+    public void execute_validClientDetails_matchingClientDetails() {
```

Comment:
For the method name perhaps it would be better to follow the convention: `whatIsBeingTested_descriptionOfTestInputs_expectedOutcome`